src/domain/event/editButtonPanel/EditButtonPanel.tsx

```diff
@@ -24,14 +25,18 @@ import useEventOrganizationAncestors from '../hooks/useEventOrganizationAncestor
 import { copyEventToSessionStorage, getEditButtonProps } from '../utils';
 import styles from './editButtonPanel.module.scss';
 
-type ActionButtonProps = { variant: ButtonVariant } & MenuItemOptionProps;
+type ActionButtonProps = {
+  isSaving: boolean;
+  variant: ButtonVariant;
```

Comment:
Does this component support  supplementary variant? If not, then this could be `Exclude<ButtonVariant, 'supplementary'>` and no need to do casting